ludwig/visualize.py

```diff
@@ -36,45 +36,88 @@
 from ludwig.utils.print_utils import logging_level_registry
 
 
-def learning_curves(
-        training_statistics,
-        field,
-        model_names=None,
-        output_directory=None,
-        file_format='pdf',
-        **kwargs
-):
-    if len(training_statistics) < 1:
-        logging.error('No training_statistics provided')
+def load_data_for_viz(training_statistics,**kwargs):
```

Comment:
I plan to do so. Maybe it will be an handler function that will call specific loaders for each type of data for visualisation and return it to the argparser before it is been sent to the visualisation function. 

I expect to have more clear design once I start adapting the rest of the visualisation functions.

Reply:
Makes sense. Probably trying to load everything and returning `None` for the missing ones and then letting the code in `cli()` pass the loaded files it needs to the actual visualization function is a good idea to start with, but yeah, you'll figure out by doing it.